src/structures/Utils.ts

```diff
@@ -213,7 +252,8 @@ export abstract class Structure {
     name: K,
     extender: (target: Extendable[K]) => T
   ): T {
-    if (!structures[name]) throw new TypeError(`"${name} is not a valid structure`);
+    if (!structures[name])
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585532)
```suggestion
    if (!structures[name])
```